charts/data-service/values.yaml

```diff
@@ -21,12 +21,9 @@ service:
 image:
   name: data-service
   base: jre
-  tag: RELEASE-955749b
+  tag: SNAPSHOT-8dfadc8
   pullPolicy: IfNotPresent
-  gitHash: 955749b
-  versionNumber: 0.5.0
-  codeRelease: 0.5.0-RELEASE
-  revision: RELEASE
+  codeRelease: 0.5.0-SNAPSHOT
```

Comment:
Not sure how this affects others pulling the code from GitHub for the services, but I do know that these variables do need to be there in some shape or form when it comes time to build and run the examples.